src/components/DashboardHome/DashContent.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
```

Comment:
Are you using standard.js to format code? Everyone else (I think) is using no semi-colon at the end of lines. Can we update these files to follow that standard?

Reply:
Done!